src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,68 @@
+.stopwatch {
+  width: 400px;
+  height: 400px;
+  border: 15px solid $main-color;
+  border-radius: 100%;
+  background-color: $background-color;
+
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin: 40px auto;
+
+  &__center {
+    width: 30px;
+    height: 30px;
+    background-color: $main-color;
+    border-radius: 100%;
+    z-index: 3;
+  }
+
+  &__sec {
+    width: 10px;
+    height: 200px;
+    background-color: $sec-hand-color;
+    top: 0;
```

Comment:
```suggestion
    bottom 40vmin;
```

Reply:
also for both arrows set `animation-iteration-count: infinite;` for make more realistic watch.
now arrows stop after finish duration.